README.md

```diff
@@ -1 +1,216 @@
+
+
+```sh
+ npm install node
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd your-folder
+  git clone https://github.com/AyoMoses1/Portfolio-Redefined.git
+```
+
+### Install
+
+Install this project with:
+
+
+
+```sh
+  cd my-project
+  node version 16
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+```sh
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/AyoMoses1)
+- Twitter: [@twitterhandle](https://twitter.com/Ayo_Moses1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ayo-moses-493946184/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please support by contributing to the codebase
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank The entire Microverse team for the well documented and highly informative resources that helped me in building this project. Thanks!!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+- **Will there be a desktop version of this project**
+
+  - Yes, there will certainly be a desktop version to aid desktop users in having a friendly UX experience on the website
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] The link to the _**LICENSE**_ is broken. That is because there is no LICENSE file in your root directory. Kindly add a LICENSE file. 👍